science/mmtf-cpp/Portfile

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.0
```

Comment:
please use the updated `cmake 1.1` PG

Reply:
How can I make this edit from the web interface below?

https://github.com/macports/macports-ports/pull/7199

Reply:
I have no idea how to do this from the we interface